packages/resolver/lib/resolver.ts

```diff
@@ -30,37 +22,30 @@ export class Resolver {
       ["working_directory", "contracts_build_directory", "contracts_directory"]
     );
 
-    resolverOptions = {
-      ...defaultResolverOptions,
-      ...resolverOptions
-    };
-    const {
-      includeTruffleSources,
-      translateJsonToSolidity,
-      resolveVyperModules
-    } = resolverOptions;
+    const { includeTruffleSources } = resolverOptions;
 
     this.options = options;
-    this.sources = [
+
+    let basicSources: ResolverSource[] = [
       new EthPMv1(options.working_directory),
       new NPM(options.working_directory),
       new GlobalNPM(),
       new FS(options.working_directory, options.contracts_build_directory)
     ];
-
     if (includeTruffleSources) {
-      this.sources.unshift(new Truffle(options));
+      basicSources.unshift(new Truffle(options));
     }
 
-    if (translateJsonToSolidity) {
-      this.sources = [].concat(
-        ...this.sources.map(source => [new ABI(source), source])
-      );
-    }
+    //set up abi-to-sol resolution
+    this.sources = [].concat(
```

Comment:
So now all sources will be "abi-to-sol"-ified whereas before it was optional? What cases were there before where we wouldn't want this to be the case?

Reply:
Well, before it was on by default, but could be turned off; specifically, `compile-vyper` would turn it off.  We didn't want it on for Vyper.  Now instead, the `ABI` resolver source will check that you're using `solc`.  So the source will be *there* when using Vyper, it just won't *do* anything.  Same as how now, the `Vyper` resolver source will be there when you use Solidity, but it won't do anything.